<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorToolWidget</class>
 <widget class="QWidget" name="ColorToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorToolWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RGB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRgb"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>回车键转换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHex"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="laColor">
       <property name="text">
        <string>颜色</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="magnifyLabel">
       <property name="text">
        <string>放大区</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbPickColorStart">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>开始拾色(Ctrl+S)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPickColorStop">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>停止拾色(Ctrl+D)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
